src/fromager/vendor_rust.py

```diff
@@ -93,7 +93,15 @@ def vendor_rust(
     ``False``.
     """
     # check for Cargo.toml
-    manifests = list(project_dir.glob("**/Cargo.toml"))
+    manifests = []
+    for manifest in project_dir.glob("**/Cargo.toml"):
+        if any((d in manifest.parts) for d in ["22 cargo subproject", "25 cargo lock"]):
```

Comment:
Can we do something that's less tied to the specific package we're trying to build?

For example, patch these files in the upstream build? Or catch errors with the vendoring and ignore them by default in fromager? 

I opened #224 to track other ideas.

Reply:
I have created https://github.com/python-wheel-build/fromager/pull/233 with a different approach. `vendor_rust` detects whether a project uses maturin or setuptools-rust. Meson and NumPy don't have a build dependency on any of the project and vendoring is skipped.

Reply:
I like that better, it's less tightly coupled to specific packages and based more on general configuration.